features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/RoomDetailsView.kt

```diff
@@ -163,13 +166,27 @@ fun RoomDetailsView(
 }
 
 @Composable
-internal fun RoomShareSection(onShareRoom: () -> Unit, modifier: Modifier = Modifier) {
-    PreferenceCategory(modifier = modifier) {
-        PreferenceText(
-            title = stringResource(R.string.screen_room_details_share_room_title),
-            icon = Icons.Outlined.Share,
-            onClick = onShareRoom,
-        )
+internal fun MainActionsSection(onShareRoom: () -> Unit, modifier: Modifier = Modifier) {
+    Row(modifier.fillMaxWidth(), horizontalArrangement = Arrangement.Center) {
+        MainActionButton(title = stringResource(R.string.screen_room_details_share_room_title), icon = Icons.Outlined.Share, onClick = onShareRoom)
+    }
+}
+
+@Composable
+internal fun MainActionButton(title: String, icon: ImageVector, onClick: () -> Unit, modifier: Modifier = Modifier) {
```

Comment:
This look like a design component, maybe extract to the design module in package `io.element.android.libraries.designsystem.components`?